File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/domain/response/TvShowResponse.java
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.domain.response;

import java.util.ArrayList;

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.domain.TvShow;

public class TvShowResponse
{
  private int id;
  private String name;
  private String first_air_date;
  private String overview;
  private float vote_average;
  private String poster_path;
  private ArrayList<TvShow> results = null;

  public TvShow getTvShow()
  {
    return new TvShow(id, name, first_air_date, overview, vote_average,
        poster_path);
  }

  public ArrayList<TvShow> getResults()
  {
    return results;
  }
}

File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/model/domain/response/media/tv_responses/TvShowResponse.kt
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.response.media.tv_responses

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.TvShow
import java.util.ArrayList

class TvShowResponse
{
    val id : Int = 0
    val name : String = ""
    val first_air_date : String = ""
    val overview : String = ""
    val vote_average : Float = 0.0F
    val poster_path : String = ""
    val results : List<TvShow> = ArrayList()

    fun getTvShow() : TvShow
    {
        return TvShow(id, name, first_air_date, overview, vote_average,
            poster_path)
    }
}